Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -2001,6 +2001,12 @@ script:
       name: category
       required: false
       secret: false
+    - default: false
+      description: A profile setting group.
+      isArray: false
+      name: profile_setting
```

Comment:
is this like a free text input or a multi select in the product? if a multi select, how dose the user get the correct values to use?

Reply:
free text, per the object he will have to create(manually from the UI)
as same as address groups objects, etc